include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -27,6 +27,10 @@
 #define ONEDPL_VERSION_MINOR 7
 #define ONEDPL_VERSION_PATCH 1
 
+#if __cplusplus < 201703L
```

Comment:
seems like everybody agrees that this message is useful. To maximize the changes that user will see this error first I would propose to move it to `common_config` since the latter is available in every public header and goes first.

Reply:
Fixed.